ehb_service/apps/api/views/externalrecord.py

```diff
@@ -255,22 +254,23 @@ def put(self, request):
                         }
                     )
 
-            return json.dumps(response)
+            return Response(response)
 
     def delete(self, request, pk):
         try:
             er = ExternalRecord.objects.get(pk=pk)
             er.delete()
-            return HttpResponse(status=codes.ok)
+            return Response(status=status.HTTP_204_NO_CONTENT)
 
         except ExternalRecord.DoesNotExist:
             log.error("Unable to delete ExternalRecord as it does not exist")
-            return HttpResponse(status=codes.not_found)
+            return Response(status=status.HTTP_404_NOT_FOUND)
 
 
-class ExternalRecordLabelResource(Resource):
+@permission_classes((permissions.AllowAny,))
```

Comment:
Removed the @permissionsAllowAny header from all the classes. Created a api/authentication.py file to override Django authentication so that we can continue authenticating using the api tokens that we've created instead of switching over to Django-generated auth-tokens. 803fea7